### DISCUSSION TITLE
Error: Failed to delete project: 401

### DISCUSSION THREAD

**Discussion by jonasschmahl** [AntlerVC/firetable#906] [Support / Q&A] [answered]
I tried to delete a project

**Comment by BeeBombshell** [comment, marked as answer]
Hey @jonasschmahl, I'm wondering why this isn't working for you. Perhaps you could email us at support@rowy.io with more information so we can assist you.

**Comment by theAPIhouse** [comment]
I have the same problem

**Reply by BeeBombshell** [reply]
Hey @KhanyaConnect, seems like many people are running into this issue. Could you explain what steps led you to this? Also, write a mail to support@rowy.io so that we can fix this for you.

**Comment by pcxnet** [comment]
Also can't delete project, have emailed support 2 weeks ago with no response.  Can't continue using it until this is fixed.

**Reply by BeeBombshell** [reply]
Hi @pcxnet, I have replied to your discussion with a direct link to set up a support call. Hope this helps.
